.github/workflows/dotnet-ec2-asg-e2e-test.yml

```
@@ -0,0 +1,296 @@
+## Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+## SPDX-License-Identifier: Apache-2.0
+
+# This is a reusable workflow for running the Dotnet E2E Canary test for Application Signals.
+# It is meant to be called from another workflow.
+# Read more about reusable workflows: https://docs.github.com/en/actions/using-workflows/reusing-workflows#overview
+name: Application Signals Enablement E2E Testing - Dotnet EC2 Asg Use Case
```

Rename to match other tests
```suggestion
name: DotNet EC2 ASG Use Case
```

---

Do the same in the other tests as well please

---

Done